src/hotspot/share/utilities/stringUtils.hpp

```diff
@@ -27,7 +27,10 @@
 
 #include "memory/allStatic.hpp"
 
-#include <string.h>
+#ifdef _WINDOWS
+  // strtok_s is the Windows thread-safe equivalent of POSIX strtok_r
```

Comment:
Nit: indent needs fixing

Reply:
I'm not seeing the indent problem... Maybe my eyes are tired...

Reply:
@dholmes-ora  Do you mean to remove 2 spaces before `//`?

Reply:
This code is copy-pasted from os.hpp.

Reply:
I am pushing it as it is since I got 2 reviews.